qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/1.1.send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: 1.1.Send CV
+
+  Scenario: 1.1_01 positive sending CV
+  Given user opened the form "Send CV"
+  When user enters valid data in the field "Name"
+  And user enters valid data in the field "Email"
+  And user attaches CV
+  And user ticked "I agree to the personal data processing"
+  And user click button "Send"
```

Comment:
And user clicks (or pushes) the "Send" button

Reply:
ok